opengrok-indexer/src/main/java/org/opengrok/indexer/search/context/HistoryContext.java

```diff
@@ -82,7 +82,12 @@ public boolean getContext(String filename, String path, List<Hit> hits) throws H
             return false;
         }
         File f = new File(filename);
-        return getHistoryContext(HistoryGuru.getInstance().getHistory(f), path, null, hits, null);
+        History history = HistoryGuru.getInstance().getHistory(f);
+        if (history == null) {
+            LOGGER.log(Level.INFO, "Null history got for {0}", f);
```

Comment:
this is perhaps a warning or HistoryException throw

Reply:
I patterned it after the other method: https://github.com/oracle/opengrok/blob/c77154d054b61f6a49a3ed10497bafb7d5385214/opengrok-indexer/src/main/java/org/opengrok/indexer/search/context/HistoryContext.java#L116-L121

Reply:
I see, then it's probably ok.